fastlane/lib/fastlane/actions/supply.rb

```diff
@@ -7,11 +7,13 @@ def self.run(params)
 
         FastlaneCore::UpdateChecker.start_looking_for_update('supply') unless Helper.is_test?
 
-        all_apk_paths = Actions.lane_context[SharedValues::GRADLE_ALL_APK_OUTPUT_PATHS] || []
-        if all_apk_paths.length > 1
-          params[:apk_paths] ||= all_apk_paths
-        else
-          params[:apk] ||= Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
+        unless params[:apk_paths].present? || params[:apk].present?
```

Comment:
I think the change is OK, but we should really consider
* adding tests. There are too many ways to set up the paths.
* go towards removing the use of `params[:apk]` at least internally. For example we could do something like

```ruby
params[: apk_paths] = [ Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH] ] if Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
```